packages/runtime/test/integration/close-clear-ssc.ts

```diff
@@ -54,11 +54,13 @@ describe("ASC - CloseOut from Application and Clear State", function () {
 
   it("should successfully closeOut from app and update state according to asc", function () {
     const appId = runtime.addApp(flags, {}, approvalProgram, clearProgram); // create app
-    closeOutParams.appId = appId;
-    runtime.optInToApp(john.address, appId, {}, {}); // opt-in to app (set new local state)
+    const initialJohnMinBalance = runtime.getAccount(john.address).minBalance;
 
-    runtime.executeTx(closeOutParams);
+    runtime.optInToApp(john.address, appId, {}, {}); // opt-in to app (set new local state)
+    syncAccount();
+    assert.isAbove(john.minBalance, initialJohnMinBalance); // verify minimum balance raised after optIn
```

Comment:
please update it to check how much it is increased.

Reply:
thought so initially, but then realized it doesn't make sense to compare constant increase (as we will be testing increment for exactly same value as we'll be increasing in code (..does that make sense?) ). Didn't do it here because we are testing min_balance in multiple flow (unlike ASA tests), which could be hard to interpret.
The important part is that minimum balance should grow after app/asset creation or opt-in, and reduce (to prev value) after opt-out/deletion.
If you want i can update to check exact value

Reply:
agree this may not make sense now, but at some later point if storage costs are changed in code, then these tests will fail and we will know why

Reply:
makes sense